cmd/api/src/docs/json/paths/v2/container-entity.json

```diff
@@ -0,0 +1,104 @@
+{
+    "/api/v2/containers/{object_id}": {
+        "parameters": [
+            {
+                "type": "string",
+                "description": "Container Object ID",
+                "name": "object_id",
+                "in": "path",
+                "required": true
+            }
+        ],
+        "get": {
+            "description": "Get basic info and counts for this Container",
+            "tags": [
+                "Container Entity API",
+                "Community",
+                "Enterprise"
+            ],
+            "summary": "Get container entity info",
+            "parameters": [
+                {
+                    "$ref": "#/definitions/parameters.PreferHeader"
+                }
+            ],
+            "responses": {
+                "200": {
+                    "description": "OK",
+                    "schema": {
+                        "$ref": "#/definitions/api.BasicResponse"
+                    }
+                },
+                "400": {
+                    "description": "Bad Request",
+                    "schema": {
+                        "$ref": "#/definitions/api.ErrorWrapper"
+                    }
+                },
```

Comment:
This could use a section for a 404 Not Found status since it is defined on line 40 in container.go